Things.md

```diff
@@ -104,14 +106,41 @@ FSM.slideToX(thing, 10, 5);
 // Slides a Thing to y position 10, moving a max distance of 5. 
 FSM.slideToY(thing, 10, 5);
 ```
-
+  
 ## Opacity  
 Opacity defines how transparent a Thing is. It's defined in the `opacity` property and ranges from 0 to 1 (0 = fully transparent, 1 = fully visible).  
-You can set a Thing's opacity using `setOpacity`.
+You can set a Thing's opacity using `setOpacity`.  
 ```javascript
 // Sets a Thing's opacity to 1. 
 FSM.setOpacity(thing, 1);
 
 // Sets a Thing's opacity to 0 (transparent). 
 FSM.setOpacity(thing, 0);
-```
\ No newline at end of file
+```
+  
+# Adding Things  
+You're able to add Things to the game via `addThing`, specifying the `x` and `y` positions (again this is relative to the top left corner of the screen). The `x` is where the Thing's `left` bound will be and the `y` is where the Thing's `top` bound will be. You can add it using the class title for the Thing:  
+```javascript
+// Adds a Thing to the game at position (1,2) using the class name.
+FSM.addThing("CastleFireball", 1, 2);
+
+// Adds a Thing to the right of another Thing.
+FSM.addThing("CastleFireball", otherThing.right, otherThing.top);
+
+// Adds a Thing below another Thing.
+FSM.addThing("CastleFireball", otherThing.left, otherThing.bottom);
+```
+  
+# TimeHandlr  
+TimeHandlr is a module used for setting timed events. It's a more flexible alternative to JavaScript's `setTimeout` and `setInterval`. It schedules events in time slots that are handled at each game upkeep (16 ms). You can also use it to add your own events:
+```javascript
+// Sets a Thing to shift right by 2 after 100 game upkeeps.
+FSM.addEvent(FSM.shiftHoriz, 100, thing, 2);
+```
+In the above example, the first two arguments to `addEvent` specify what function to call and after how many milliseconds, respectively. Any arguments passed to addEvent after these two are passed to the callback function (the first argument). In terms of the above example, `thing` and `2` are passed as the arguments to `shiftHoriz` when it's called in 100 milliseconds.  
```

Comment:
You should say how a function works before an example, then explain how the example correlates to the functionality.

Right now you're giving an example, *then* explaining how the function works and how the example correlates.